var/spack/repos/builtin/packages/of-catalyst/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class OfCatalyst(CMakePackage):
+    """Of-catalyst is a library for OpenFOAM that provides a runtime-selectable
+    function object for embedding ParaView Catalyst in-situ visualization
+    into arbitrary OpenFOAM simulations.
+    Supports in-situ conversion of the following types:
+      1) finite volume meshes and fields, single or multi-region;
+      2) finite area meshes and fields, single region;
+      3) lagrangian (clouds), single or multiple clouds.
+    This offering is part of the community repository supported by OpenCFD Ltd,
+    producer and distributor of the OpenFOAM software via www.openfoam.com,
+    and owner of the OPENFOAM trademark.
+    OpenCFD Ltd has been developing and releasing OpenFOAM since its debut
+    in 2004.
+    """
+
+    # Currently only via git
+    homepage = "https://develop.openfoam.com/Community/catalyst"
+    gitrepo  = "https://develop.openfoam.com/Community/catalyst.git"
```

Comment:
If you name this `git`, Spack now supports package-level git tags, so you won't need to specify it in `version()` below.

Reply:
Sorry, name what with git?

Reply:
```suggestion
    git  = "https://develop.openfoam.com/Community/catalyst.git"
```

Reply:
done